README.md

```diff
@@ -827,12 +827,12 @@ Returns: `string`
 
 Usage:
 ```
-{{currency 1000000 code='USD'}}  => $1,000,000.00
-{{currency 1000000 code='EUR'}}  => 1 000 000,00 €
-{{currency 1000000 code='EUR' precision=0}}  => 1 000 000 €
+{{formatCurrency 1234567.89 currency='USD'}}  => $1,234,567.89
```

Comment:
We can do this but, we may have to define the parameters explicitly in README with the reference because of change in the parameter's name.